src/Discord.Net.Commands/Info/CommandInfo.cs

```diff
@@ -63,6 +63,9 @@ internal CommandInfo(CommandBuilder builder, ModuleInfo module, CommandService s
             _action = builder.Callback;
         }
 
+        public TPrecondition GetPrecondition<TPrecondition>() where TPrecondition : PreconditionAttribute =>
```

Comment:
Yeah, I don't know of any case where someone would have duplicate preconditions on a command. As for `x => x is TPrecondition`, see earlier in the PR conversation. That would cause undesirable behavior with preconditions that derive from other preconditions.